.github/workflows/push-mpc-to-ecr.yml

```diff
@@ -0,0 +1,51 @@
+name: Build & Push MPC Signer to Amazon ECR
+
+on:
+  push:
+    branches: [ "matt/feature/mpc-signer" ]
```

Comment:
Todo: Change this to the appropriate branch for CICD.

Reply:
Thanks @MattSenter for the PR!
That's great.
Let's change the branch to "feature/mpc-signer" to use this functionality right away.
Also I suggest to add "on: workflow_dispatch" event to trigger it manually or additional trigger on the merged pull request